### PR TITLE
Update compass chart after introducing integration system for KEB

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Updated compass chart after introducing an integration system for KEB in the PR: https://github.com/kyma-incubator/compass/pull/635

**Related issue(s)**

https://github.com/kyma-incubator/compass/pull/635

### CODE REVIEW

File: resources/compass/charts/kyma-environment-broker/templates/deployment.yaml [hunk 1]
```diff
@@ -16,6 +16,17 @@ spec:
...
+      {{ if .Values.global.isLocalEnv }}
+      hostAliases:
+        - ip: {{ .Values.global.minikubeIP }}
+          hostnames:
+            - "{{ .Values.global.gateway.tls.host }}.{{ .Values.global.ingress.domainName }}"
+            - "{{ .Values.global.gateway.mtls.host }}.{{ .Values.global.ingress.domainName }}"
+            - "{{ .Values.global.gateway.tls.secure.oauth.host }}.{{ .Values.global.ingress.domainName }}"
+            - "{{ .Values.global.oauth2.host }}.{{ .Values.global.ingress.domainName }}"
+      {{ end }}
+      serviceAccount: {{ include "kyma-env-broker.fullname" . }}
```
Comment (mszostok) [inline resources/compass/charts/kyma-environment-broker/templates/deployment.yaml]: the `serviceAccount` field is deprecated, please remove that

File: resources/compass/charts/kyma-environment-broker/templates/deployment.yaml [hunk 2]
```diff
@@ -16,6 +16,17 @@ spec:
         app.kubernetes.io/name: {{ include "kyma-env-broker.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      {{ if .Values.global.isLocalEnv }}
+      hostAliases:
```
Comment (mszostok) [inline resources/compass/charts/kyma-environment-broker/templates/deployment.yaml]: could you add a comment why we need that and why all of those hostnames and not only some of them?

**Comment by polskikiel** [comment]
@mszostok I've added `trap kill_proxy_and_exit ERR EXIT` at the beginning of the job and fixed your comments.

File: resources/compass/charts/kyma-environment-broker/templates/deployment.yaml (hunk 1 above)
Comment (polskikiel) [reply resources/compass/charts/kyma-environment-broker/templates/deployment.yaml]: removed

**Pull request merged**